vnext/ReactUWP/Modules/NativeUIManager.h

```diff
@@ -58,6 +58,10 @@ class NativeUIManager : public facebook::react::INativeUIManager
   // this function iterates all roots and try to get a valid XamlRoot.
   winrt::XamlRoot tryGetXamlRoot();
 
+  // Searches itself and its parent to get a valid XamlView.
+  // Like Mouse/Keyboard, the event source may not have matched XamlView.
+  XamlView reactPeerOrContainerFrom(winrt::FrameworkElement fe);
```

Comment:
Have we decided to add descriptions for functions in the headers? I prefer to get the overview without this, but mostly I want us to be consistent here and not just start introducing this randomly in some spots. That gives us the worst of both worlds. I realize a few other check-ins have done this (like for tryGetXamlRoot() above).

Reply:
I prefer put it in header. It has nice tips when your mouse hove over the function. but if you put it in cpp, you can't see it.

![image](https://user-images.githubusercontent.com/6290692/59801949-5b40b980-929d-11e9-9dd8-1d5103f77b07.png)